addon/adapters/github.js

```diff
@@ -19,7 +19,8 @@ export default RESTAdapter.extend({
   }),
 
   pathForType(type) {
-    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')));
+    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')))
+      .replace('organizations', 'orgs');
```

Comment:
What do you think of just implementing `pathForType` in orgs to return a static string?